decompressed/gui_file/www/lang/it-it/web-framework-tch.po

```diff
@@ -285,66 +285,66 @@ msgid "show advanced"
 msgstr "Visualizza opzioni avanzate"
 
 msgid "Invalid length, a WEP key must be 5, 10, 13 or 26 characters long, length of 10 and 26 can only contain the letters A to F or digits"
-msgstr ""
+msgstr "Lunghezza chiave non valida.<br>Una chiave WEP dever avere una lunghezza di 5 o 10 o 13 o 26 caratteri.<br>Una chiave di lungehzza 10 o 26 caratteri può contenere solo le lettere dalla A alla F o numeri."
 
 msgid "Cannot use the broadcast address"
-msgstr ""
+msgstr "Impossibile usare un indirizzo broadcast"
 
 msgid "Cannot use the limited broadcast destination address"
-msgstr ""
+msgstr "Impossibile usare l'indirizzo destinazione broadcast limitato"
 
 msgid "OFF"
-msgstr ""
+msgstr "OFF"
 
 msgid "A WEP key of length 10 or 26 can only contain the letters A to F or digits"
-msgstr ""
+msgstr "Una chiave WEP di lunghezza 10 or 26 caratteri può contenere solo lettere dalla A alla F o numeri"
 
 msgid "SSID should not start with ; # !"
-msgstr ""
+msgstr "L?SSID non può iniziare con ; # !"
 
 msgid "Invalid value; enter 'infinite' for infinite time or a number greater than 0, followed by 's' for seconds or 'm' for minutes or 'h' for hours or 'd' for days or 'w' for weeks. No spaces."
-msgstr ""
+msgstr "Valore non valido.<br>Inserisci 'infinite' per un tempo infinito, o un numero maggiore di 0, seguito da 's' per i secondi, 'm' per i minuti, 'h' per le ore, 'd' per i giorni o W per le settimane. Senza spazi."
 
-msgid "SSID should not contain ?, \, $, [, \\, ],and + special characters"
-msgstr ""
+msgid "SSID should not contain ?, \\, $, [, \\, ],and + special characters"
+msgstr "Il valore SSID non può contenere i caratteri ?, \\, $, [, \\, ], o caratteri speciali"
 
 msgid "ON"
-msgstr ""
+msgstr "ON"
 
 msgid "space is not allowed"
-msgstr ""
+msgstr "spazio non permesso"
 
 msgid "Cannot use a multicast address"
-msgstr ""
+msgstr "Impossibile usare un indirizzo multicast"
 
 msgid "Cannot use the network address"
-msgstr ""
+msgstr "Impossibile usare l'indirizzo di rete"
 
 msgid "Invalid netmask"
-msgstr ""
+msgstr "Maschera di rete non valida"
 
 msgid "SSID should not be empty"
-msgstr ""
+msgstr "L'SSID non può essere vuoto"
 
-msgid "The wireless key contains invalid characters, only space, letters, numbers and the following characters !\'`#$%&()*+,-./:;<=>?@[\\]^_{|}~ are allowed."
-msgstr ""
+msgid "The wireless key contains invalid characters, only space, letters, numbers and the following characters !'`#$%&()*+,-./:;<=>?@[\\]^_{|}~ are allowed."
+msgstr "La chiave wireelss contiene caratteri non validi.\vSono permessi solo spazi, lettere, numeri e i seguenti caratteri !'`#$%&()*+,-./:;<=>?@[\\]^_{|}~."
 
 msgid "%dm"
 msgid_plural "%dm"
-msgstr[0] ""
-msgstr[1] ""
+msgstr[0] "%dm"
+msgstr[1] "%dm"
 
 msgid "%dh"
 msgid_plural "%dh"
-msgstr[0] ""
-msgstr[1] ""
+msgstr[0] "%dh"
```

Comment:
immagino sia "o" di ore

Reply:
Immaginavio che fosse una variabile %dh.
Se la variabile è %d e la ho dopo indica ore la traduco in o (ore)

Reply:
la variabile e' appunto solo %d quello dopo sono i riferimenti del tempo